rules.ini

```
@@ -241,6 +241,8 @@ Lua[] = \.lua$
 Miles_Sound_System = (?:^|/)mss(?:16|32|64)\.dll$
 Modio = (?:^|/)mod\.?io
 NodeJS = (?:^|/)node\.dll$
+NodeJS[] = (?:^|/)node\.dll$
+NodeJS[] = (?:^|/)libnode\.(?:dylib|so)$
```

```suggestion
NodeJS = (?:^|/)(?:lib)?node\.(?:dll|dylib)$
```
unless you have an example of it's usage only on linux, let's not add it